.github/workflows/ci.yml

```diff
@@ -37,12 +37,6 @@ jobs:
         ci/code_checks.sh lint
       if: always()
 
-    - name: Dependencies consistency
-      run: |
-        source activate pandas-dev
-        ci/code_checks.sh dependencies
```

Comment:
Do we check for `dependencies` now anywhere? If not should just remove from code_checks.sh altogether too

Reply:
It's checked in pre-commit.

Reply:
Clarification - I am asking if the `dependencies` argument is used anywhere. I don't think it is but has relics of it in code_checks (?)

Reply:
Sorry... misunderstanding. It's not used anywhere else. And I just removed related lines from code_check.sh.